### ISSUE TITLE
[PAGE] Search results

### ISSUE THREAD

**Issue by vakatastz** [chingu-voyage6/toucans-team-22#6]
Hitting Enter on the search textbox (on any page) leads to Search results page? Opinions?

**Comment by AlexandraTod** [comment]
Yes, since that would be in the navbar, from what I can recall from the June meeting.
